plugin/completion/bin_complete.py

```diff
@@ -168,7 +168,7 @@ def init(self, view, settings):
         self.flags_dict[view.buffer_id()] = clang_flags
         log.debug(" clang flags are: %s", self.flags_dict[view.buffer_id()])
 
-    def complete(self, view, cursor_pos, show_errors):
+    def complete(self, view, cursor_pos, current_job_id):
```

Comment:
Outdated documentation for new argument.

BTW. Maybe you should consider only having documentation in the base class for overriding methods? Repetition is not typically good and also it's to easy for things to go out of sync.